docs/README.txt

```diff
@@ -0,0 +1,10 @@
+How do I generate the docs?
+###########################
```

Comment:
This is documented here:
http://treeherder.readthedocs.io/common_tasks.html#building-the-docs-locally